Extensions/ParticleSystem/particleemitterobject-pixi-renderer.ts

```diff
@@ -420,6 +420,25 @@ namespace gdjs {
       return this.started;
     }
 
+    /**
+     * @returns `true` at the end of emission or at the start if its paused
+     * or never if there is no limit.
+     */
+    _mayHaveEndedEmission(): boolean {
```

Comment:
I'm confused by the name of this function. 
Should this be more `_hasEndedEmission` ?
`_may` gives the impression that you can't trust the result of this function

Reply:
It's my intent. It's impossible to simply check if the emission has ended because of the way the particle emitter lib works.
These are some conditions that are needed but the result doesn't mean anything without checking the object state.